react/features/app-settings/components/native/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as BackButton } from './BackButton';
+export { default as FormRow } from './FormRow';
+export { default as FormSectionHeader } from './FormSectionHeader';
```

Comment:
All these components looks pretty generic, aren't they? How about having them in features/mobile/forms or something similar, so we can eventually reuse them?

Reply:
I agree, but let's do this in another PR. I don't really want to further grow this PR, if not necessary. Then I'll address this, and the introduction of safe areas (your previous comment) on the conference screen in separate commits, if that's okay.

Also, @lyubomir what do you think?